splunk-apm/splunk-apm.md

```diff
@@ -1,22 +1,22 @@
 <h3>Splunk OpenTelemetry Lambda Layer</h3>
 
 <pre>
-arn:aws:lambda:us-east-2:254067382080:layer:splunk-apm:24
-arn:aws:lambda:us-east-1:254067382080:layer:splunk-apm:24
-arn:aws:lambda:eu-central-1:254067382080:layer:splunk-apm:24
-arn:aws:lambda:us-west-1:254067382080:layer:splunk-apm:24
-arn:aws:lambda:us-west-2:254067382080:layer:splunk-apm:24
-arn:aws:lambda:ap-south-1:254067382080:layer:splunk-apm:24
-arn:aws:lambda:ap-southeast-1:254067382080:layer:splunk-apm:24
-arn:aws:lambda:ap-southeast-2:254067382080:layer:splunk-apm:24
-arn:aws:lambda:ap-northeast-1:254067382080:layer:splunk-apm:24
-arn:aws:lambda:ap-northeast-2:254067382080:layer:splunk-apm:24
-arn:aws:lambda:ca-central-1:254067382080:layer:splunk-apm:24
-arn:aws:lambda:eu-west-1:254067382080:layer:splunk-apm:24
-arn:aws:lambda:eu-west-2:254067382080:layer:splunk-apm:24
-arn:aws:lambda:eu-west-3:254067382080:layer:splunk-apm:24
-arn:aws:lambda:eu-north-1:254067382080:layer:splunk-apm:24
-arn:aws:lambda:sa-east-1:254067382080:layer:splunk-apm:24
+arn:aws:lambda:us-east-2:254067382080:layer:splunk-apm:28
```

Comment:
@Kielek spotted that here we have `28` while in all other places it is `27`. Is it intended?